README

```diff
@@ -82,7 +82,7 @@
 
  In order to avoid spam, this is a moderated mailing list, and it might
  take a couple of days for the ticket to show up.  (We also scan posts to make
- sure that security disclosures aren't publically posted by mistake.) Mail
+ sure that security disclosures aren't publicly posted by mistake.) Mail
```

Comment:
There seems to be some debate about publically being correct or not, but I have no problem either way.